src/hooks/screen/useChatScreenHook.ts

```diff
@@ -8,11 +8,14 @@ import {v4 as uuid} from 'uuid';
 
 import AnonymousMessageRepo from '../../service/repo/anonymousMessageRepo';
 import ChatSchema from '../../database/schema/ChatSchema';
+import ImageUtils from '../../utils/image';
 import SignedMessageRepo from '../../service/repo/signedMessageRepo';
 import UseChatScreenHook from '../../../types/hooks/screens/useChatScreenHook.types';
+import UserSchema from '../../database/schema/UserSchema';
 import useChatUtilsHook from '../core/chat/useChatUtilsHook';
 import useLocalDatabaseHook from '../../database/hooks/useLocalDatabaseHook';
 import useMessageHook from './useMessageHook';
+import useUserAuthHook from '../core/auth/useUserAuthHook';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements are not sorted alphabetically. This is not a critical issue, but it's a good practice to keep imports sorted for better readability and maintainability.
```diff
- import ImageUtils from '../../utils/image';
- import SignedMessageRepo from '../../service/repo/signedMessageRepo';
- import UseChatScreenHook from '../../../types/hooks/screens/useChatScreenHook.types';
- import UserSchema from '../../database/schema/UserSchema';
- import useChatUtilsHook from '../core/chat/useChatUtilsHook';
- import useLocalDatabaseHook from '../../database/hooks/useLocalDatabaseHook';
- import useMessageHook from './useMessageHook';
- import useUserAuthHook from '../core/auth/useUserAuthHook';
+ import ImageUtils from '../../utils/image';
+ import SignedMessageRepo from '../../service/repo/signedMessageRepo';
+ import UserSchema from '../../database/schema/UserSchema';
+ import UseChatScreenHook from '../../../types/hooks/screens/useChatScreenHook.types';
+ import useChatUtilsHook from '../core/chat/useChatUtilsHook';
+ import useLocalDatabaseHook from '../../database/hooks/useLocalDatabaseHook';
+ import useMessageHook from './useMessageHook';
+ import useUserAuthHook from '../core/auth/useUserAuthHook';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->